plugins/woocommerce-admin/client/products/fields/attribute-input-field/attribute-input-field.tsx

```diff
@@ -76,11 +104,20 @@ export const AttributeInputField: React.FC< AttributeInputFieldProps > = ( {
 			getItemValue={ ( item ) => item?.id || '' }
 			selected={ value }
 			onSelect={ ( attribute ) => {
-				onChange( {
-					id: attribute.id,
-					name: attribute.name,
-					options: [],
-				} );
+				if ( attribute.id === -99 ) {
```

Comment:
To reduce the number of `attribute.id === -99` checks, maybe we could create a util method `isNewAttribute( attribute )` that handles this check for us.

Reply:
Done as part of: https://github.com/woocommerce/woocommerce/pull/35100/commits/2dcbcd50c61e66e3a28be45f78d9fc2101674947